core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -990,6 +987,14 @@ object PSBTGlobalKeyId {
     case _: Byte                       => UnknownKeyId
   }
 
+  def fromKey(key: ByteVector): PSBTGlobalKeyId = {
```

Comment:
Since we are in the `PSBTGlobalKeyId` object can't this be called `fromBytes`? Same with the other two

Reply:
At the very least it should be `fromKeyBytes`